### ISSUE TITLE
Recent upgrade replaces excellent debugger with some blurb ... I can't remove it

### ISSUE THREAD

**Issue by mcccol** [timstuyckens/chromeextensions-knockoutjs#33]
Hi.  Since the last upgrade, all I can see when I click to the knockoutjs context debugger is the enclosed screen.  There appears to be no escaping it - it's unusable.  Can you please advise?

Thanks, Colin
![screenshot from 2017-04-25 02-36-25]
(https://cloud.githubusercontent.com/assets/15832590/25349331/9418905a-2960-11e7-8fb6-030e82be041e.png)

**Comment by timstuyckens** [comment]
Hi, 
The panel is one of the tabs in the developer tools. You can just switch tabs to leave the screen.
There hasn't been a recent upgrade.

![image](https://cloud.githubusercontent.com/assets/1768681/25368577/a728094c-297d-11e7-91a5-790f6f5b3212.png)

Kind regards,

Tim.

**Comment by mcccol** [comment]
Thank you for the clarification. I was used to seeing a ko context browser in that pane instead of the html I now see. No combination of options pressed in the extension config changes the pane in any way. Colour me confused.

**Comment by mcccol** [comment]
Ok.  I have it back.  I was missing the serializer in that pane.  I failed to induce the option enabling serializer to save, for some reason, after I received an update including tracing of observables (which seemed to me to be recent.)  I made sure to deselect and then select the option which stated it was saving, and then it all worked fine.  Thanks for an excellent facility, sorry to trouble you with naive gui issues.
